Homework/src/by/tms/Lesson_7/transports/Car.java

```diff
@@ -0,0 +1,39 @@
+package by.tms.Lesson_7.transports;
+
+public class Car extends LandTransport {
+
+    private String bodyType;
+    private int countOfPassengers;
+
+    public Car(int power, int maxSpeed, int weight, String brand, int countOfWheels, int fuelConsumption,
+               String bodyType, int countOfPassengers) {
+        super(power, maxSpeed, weight, brand, countOfWheels, fuelConsumption);
+        this.bodyType = bodyType;
+        this.countOfPassengers = countOfPassengers;
+    }
+
+    public void countMaxParameters(double time) {
+        double maxWay = maxSpeed * time;
+        double fuel = litersOfFuel(maxWay);
+        System.out.println("For " + time + " hours the car " + brand + " with the max speed " + maxSpeed +
+                " km/h will pass " + maxWay + " km and will use up " + fuel + " liters of fuel.");
+    }
+
+    private double litersOfFuel(double way) {
+        return fuelConsumption * way / 100;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
It's better to call super.toString() and include response in current method.
Because if number/type/count of fields in superclass will be changed - you'll need to overwrite all toString() methods in inherited classes
apply this solution to all inherited classes